MbCache/Configuration/CacheKeyBase.cs

```diff
@@ -85,16 +87,14 @@ public KeyAndItsDependingKeys GetAndPutKey(ComponentType type, ICachingComponent
 			var scope = Scope();
 			var completeKey = scope == null ? 
 				startKey : 
-				string.Concat(startKey, separator, Scope());
+				string.Concat(startKey, separator, scope);

```

Comment:
This change I already did yesterday. Haven't touched this code base in month... What a coincidence!